classes/helper.php

```diff
@@ -371,4 +374,96 @@ public static function regular_expression_search(string $search, string $table,
         }
         return $results;
     }
+
+    /**
+     * Get column info from column name.
+     *
+     * @param string $table The table name.
+     * @param string $columnname The column name.
+     */
+    private static function get_column_info(string $table, string $columnname): database_column_info {
+        global $DB;
+
+        $columns = $DB->get_columns($table);
+        $column = null;
+        foreach ($columns as $col) {
+            if ($col->name == $columnname) {
+                $column = $col;
```

Comment:
why not simplify this and just return here and avoid maybe 5 lines of code?

Reply:
I have updated this and removed the error string.